kolibri/core/templates/kolibri/base.html

```diff
@@ -18,10 +15,11 @@
 </ul>
 {% endblock %}
 
-<hr>
-
-{% block content %}
-
+<main></main>
+{% block frontend_assets %}
```

Comment:
There isn't any HTML to parse since the UI is completely dynamically generated, so in practical terms shouldn't be any performance difference. This just seemed easier.